pkgs/development/libraries/libfprint/default.nix

```diff
@@ -1,22 +1,29 @@
-{ stdenv, fetchurl, pkgconfig, libusb, pixman, glib, nss, nspr, gdk_pixbuf }:
+{ stdenv, fetchurl, pkgconfig, meson, ninja, libusb, pixman, glib, nss, gtk3
+, libX11, libXv, coreutils, gtk-doc, docbook_xsl, docbook_xml_dtd_43 }:
 
 stdenv.mkDerivation rec {
-  name = "libfprint-0.7.0";
+  name = "libfprint-${version}";
+  version = "0.99.0";
 
   src = fetchurl {
-    url = "https://people.freedesktop.org/~anarsoul/${name}.tar.xz";
-    sha256 = "1wzi12zvdp8sw3w5pfbd9cwz6c71627bkr88rxv6gifbyj6fwgl6";
+    url = "https://gitlab.freedesktop.org/libfprint/libfprint/uploads/82ba3cef5bdf72997df711eacdb13c0f/libfprint-${version}.tar.xz";
+    sha256 = "16r4nl40y0jri57jiqmdz4s87byblx22lbhyvqpljd6mqm5rg187";
   };
 
-  buildInputs = [ libusb pixman glib nss nspr gdk_pixbuf ];
-  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ libusb pixman glib nss gtk3 libX11 libXv ];
```

Comment:
IIUC `gtk3 libX11 libXv` are required only to build X11 and GTK example applications.
But do we want to build (and ship) those examples along with the lib by default?

Reply:
@c0bw3b 
Dropped dependance on X11 libraries, but retained dependance on gtk3 since its needed to build demo.
`fprint_demo` package is obsoleted and replaced by `${pkgs.libfprint}/bin/gtk-libfprint-test` which is useful to test whether sensor works correctly and how to exactly scroll finger.